packages/fx-core/tests/plugins/resource/simpleauth/unit/expectedBicepFiles/onlyWithSimpleAuthPlugin/simple_auth_test1.bicep

```diff
@@ -0,0 +1,57 @@
+param sku string
```

Comment:
why we append `1` to the file name?

Reply:
Just to differentiate two test files. Changed to same file name.